src/font/sprite/Box.zig

```diff
@@ -25,6 +25,241 @@ const z2d = @import("z2d");
 const font = @import("../main.zig");
 const Sprite = @import("../sprite.zig").Sprite;
 
+const OctantMin = 0x1cd00;
+const OctantMax = 0x1cde5;
+const Octants = [OctantMax - OctantMin + 1][8]bool{
+    [_]bool{ false, false, true, false, false, false, false, false },
```

Comment:
It seems rather tedious and error-prone to convert the name of the octant from "BLOCK OCTANT-12345" to a boolean array like this. With comptime strings and inline for loops this can be made so much easier.

Reply:
hrmmm, i agree that using the language's native features for this kind of thing would be far less opaque. i'm intrigued, but doubt i'll get to it this weekend--feel encouraged to use this PR as a starting point to do something better. i have no emotional ties to this matrix.

Reply:
thanks for the pointers to the ziggery, though!

Reply:
I haven't reviewed this PR yet but I agree this code is very C (no surprise!). Zig's comptime facilities should allow us to do something much nicer here I believe.

Reply:
Looking closer, the skips in the octants are much less regular than expected, we're probably better off just deriving them from the unicode names at comptime then.